src/Application/Folders/Commands/AddFolder/AddFolderCommandValidator.cs

```diff
@@ -17,7 +17,8 @@ public AddFolderCommandValidator()
             .MaximumLength(256).WithMessage("Description cannot exceed 256 characters.");
 
         RuleFor(f => f.Capacity)
-            .GreaterThanOrEqualTo(1).WithMessage("Folder's capacity cannot be less than 1");
+            .NotEmpty().WithMessage("Folder is required.")
```

Comment:
This should have been "Folder's capacity is required"

Reply:
fixed at https://github.com/ProFile-org/ProFile-BE/pull/88/commits/76a4df4ca33e0acda026746dd9e57214d3d9e34f